packages/grafana-ui/src/components/Table/utils.ts

```diff
@@ -108,6 +114,23 @@ export function getColumns(data: DataFrame, availableWidth: number, columnMinWid
   return columns;
 }
 
+function getFooterValue(index: number, footers?: DataFrame) {
+  if (footers === undefined) {
+    return EmptyCell;
+  }
+  const field = footers.fields[index];
+  const fieldValue = field.values.get(0);
```

Comment:
rather than define the cell explictly, should we just use `getCellComponent(..)`?